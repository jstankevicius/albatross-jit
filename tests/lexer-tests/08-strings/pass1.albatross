string var := "strings constants look like this";
string str := "this is a longer string literal constant";
string sml := "and a small one";
