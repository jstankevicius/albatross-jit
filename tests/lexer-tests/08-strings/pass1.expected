1 1 TYPE string
8 1 NAME var TYPE string
12 1 ASSIGN
15 1 STRING 32 strings constants look like this
49 1 SEMICOLON
1 2 TYPE string
8 2 NAME str TYPE string
12 2 ASSIGN
15 2 STRING 40 this is a longer string literal constant
57 2 SEMICOLON
1 3 TYPE string
8 3 NAME sml TYPE string
12 3 ASSIGN
15 3 STRING 15 and a small one
32 3 SEMICOLON
